public/views/projects-migrating.html

```diff
@@ -8,13 +8,13 @@
 
 <div ng-if="migratingProjects.length && !migratingProjectsError">
 	<ul class="list-group" style="margin-bottom:0">
-  	<li class="list-group-item" ng-repeat="migratingProject in migratingProjects | orderBy:site_id">
-  		<div style="border-bottom: 1px solid #ddd;margin:-6px -13px 10px;padding:0 5px 5px 5px" ng-if ="migratingProject.site_id !== migratingProjects[$index - 1].site_id">
+  	<li class="list-group-item" ng-repeat="migratingProject in migratingProjects">
+  		<div style="border-bottom: 1px solid #ddd;margin:-6px -13px 10px;padding:0 5px 5px 5px">
```

Comment:
That is just there so that we can group (visually) the current migrations by site ID. So that we can do:

Site Name
 - migration 1
 - migration 2
 - migration 3

But since we are ordering by migration start time that is no longer possible, so am taking it out.